### ISSUE TITLE
Embedded WG meeting, July 9

### ISSUE THREAD

**Issue by japaric** [rust-lang-nursery/embedded-wg#111]
[6-7 PM CEST][0] on #rust-embedded on irc.mozilla.org

[0]: https://everytimezone.com/#2018-7-9,240,b8jj

Agenda.

- Planning work on the embedded Rust book. cf. #56
  - Final outline. Are all OK with the outline in https://book.rust-embedded.org/ ?
  - Prioritization of topics / sections. Which topics *must* be done and polished for the edition release? Which topics would be nice to have but are not critical?
  - How to have the community contribute to the book? How to commit to writing a section / about a topic? Where to send feedback about the book?

- Maintaining the embedded / Cortex-M ecosystem. cf. #46. @japaric will come up with some plan / structure.

- Triaging issues on the [2018 edition milestone](https://github.com/rust-lang-nursery/embedded-wg/milestone/1).

- [your topic goes here]

This meeting is open for anyone to attend. If you'd like to bring up any issue / topic related to embedded Rust leave a comment in this issue so we can add it to the agenda.

**Comment by jamesmunns** [comment]
Might be worth mentioning, some of the thumbv8em targets are now purchasable as chips/dev boards: https://github.com/rust-lang-nursery/embedded-wg/issues/88#issuecomment-403275615

**Comment by jamesmunns** [comment]
Finalize decision on Embedded Rust Logo: https://github.com/rust-lang-nursery/embedded-wg/issues/107#issuecomment-398579852

To-Do:

* Initial licensing guidelines (copy/paste from upstream rust?)
* Add to embedded-wg README
* Update twitter logo

**Comment by jamesmunns** [comment]
Status of the Embedded CI/IRR program:

@sekineh has opened up this issue to try and unblock our "base case" tests: https://github.com/rust-lang/rust/issues/52163

@nerdyvaishali is going to capture our steps as issues/milestones based on 3 week cycles (aligned with the compiler releases).

**Comment by japaric** [comment]
This meeting happened. Logs are [here](https://github.com/rust-lang-nursery/embedded-wg/blob/master/minutes/2018-07-09.irc.log). Issue for next meeting [here](https://github.com/rust-lang-nursery/embedded-wg/issues/123).
